src/worker/core/GameSim.basketball/PlayByPlayLogger.ts

```diff
@@ -15,80 +17,101 @@ type PlayByPlayEventInputScore =
 			pid: number;
 			pidDefense: number;
 			pidAst: number | undefined;
+			pidTotalPts: number;
+			pidAstTotalAst: number | undefined;
```

Comment:
In theory we don't need to pass the values here because stats are already being sent to the box score UI, so the UI already has all the stats for all players. Not sure how easy it is to access, but that would probably be cleaner than manually passing them in everywhere. lmk if you need help figuring that out.

And related to what I said in the other PR about game sim stuff not being well abstracted... this would be a great feature in the other sports too, but sadly will require some special work for each sport.